validation-test/Serialization/Foundation-determinism-wmo.swift

```diff
@@ -1,5 +1,5 @@
-// RUN: %target-build-swift -module-name Foundation -parse-as-library %S/../../stdlib/public/SDK/Foundation/*.swift %T/../../../stdlib/public/SDK/Foundation/8/*.swift -emit-module-path %t.1.swiftmodule -force-single-frontend-invocation
-// RUN: %target-build-swift -module-name Foundation -parse-as-library %S/../../stdlib/public/SDK/Foundation/*.swift %T/../../../stdlib/public/SDK/Foundation/8/*.swift -emit-module-path %t.2.swiftmodule -force-single-frontend-invocation
+// RUN: %target-build-swift -module-name Foundation -parse-as-library %S/../../stdlib/public/SDK/Foundation/*.swift `dirname %t`/../../../stdlib/public/SDK/Foundation/8/*.swift -emit-module-path %t.1.swiftmodule -force-single-frontend-invocation
```

Comment:
Is this particular change an improvement? It seems to be using `%T` as read-only, and the replacement seems even more fragile than `%T`.

Reply:
Is there a better way to get path to the build directory?

Btw this test is currently disabled anyway...

Reply:
And %T is going to get removed.  We have to stop using it.

Reply:
Oh, I see that `%T` is being removed entirely as a substitution upstream.